substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/management/Target_jdk_internal_agent_FileSystemImpl.java

```diff
@@ -0,0 +1,15 @@
+package com.oracle.svm.core.jdk.management;
+import com.oracle.svm.core.annotate.Substitute;
+import com.oracle.svm.core.annotate.TargetClass;
+import jdk.internal.agent.FileSystemImpl;
+
+import java.io.File;
+
+@TargetClass(FileSystemImpl.class)
+final class Target_jdk_internal_agent_FileSystemImpl {
+
+//    @Substitute
```

Comment:
I have substituted this method as a temporary work-around because it causes a linking error when using JMX with password authentication.